src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const daysForSmallDiscount = 3;
+  const daysForBigDiscount = 7;
+  const carRentPricePerDay = 40;
+  const smallDiscountPercentage = 20;
+  const bigDiscountPercentage = 50;
+  const totalRentCost = carRentPricePerDay * days;
+
+  if (days < daysForSmallDiscount) {
+    return totalRentCost;
+  } else if (days >= daysForSmallDiscount && days < daysForBigDiscount) {
```

Comment:
Do we really need `else` here? Also maybe this condition in the brackets can be simplified a bit? Think in which case we will arrive at this point in the code, if we have `return` statement before :)